### ISSUE TITLE
[CS2] Rewriter: Inline block comments in CSX not getting output

### ISSUE THREAD

**Issue by microdou** [jashkenas/coffeescript#4629]
`JSX` has a comment syntax `{/*  */}`:
```javascript
<div> {/* A JSX comment */} </div>

<div>
{/* 
  Multiline
  JSX comment
*/}
</div>
```
Currently there is no corresponding comment syntax in `CSX` yet. However, a workaround does exist, by using ```{`/*  */`}```:
```coffeescript
# CSX
<div> {`/* A JSX comment */`} </div>
<div>
{`/* 
  Multiline
  JSX comment
*/`}
</div>

# converts to JSX
<div> {/* A JSX comment */} </div>;
<div>
{/* 
  Multiline
  JSX comment
*/}
</div>;
```
Should this workaround be documented, or implement a specific comment syntax for `CSX`?
It can be `{###  ###}` or `{#  }` or something else. There was some discussion on this topic in the _depreciated `CJSX`_ https://github.com/jsdf/coffee-react-transform/pull/41, and _`CJSX`_ actually implemented the syntax `{#  }`.

**Comment by GeoffreyBooth** [comment]
Regular block comments [are allowed](http://coffeescript.org/v2/#try:%3Cdiv%3E%20%7B%23%23%23%20A%20JSX%20comment%20%23%23%23%7D%20%3C%2Fdiv%3E%0A%0A%3Cdiv%3E%0A%7B%23%23%23%0A%20%20Multiline%0A%20%20JSX%20comment%0A%23%23%23%7D%0A%3C%2Fdiv%3E) within CSX:

```coffee
<div> {### A JSX comment ###} </div>

<div>
{###
  Multiline
  JSX comment
###}
</div>
```

Though they’re not getting output at the moment.

**Comment by microdou** [comment]
@GeoffreyBooth That's great! I didn't realize that its already in `beta4`.

**Comment by GeoffreyBooth** [comment]
Fixed by #4659.
